server/sonar-web/src/main/js/app/components/nav/component/ComponentNavBranch.tsx

```diff
@@ -60,38 +64,84 @@ export default class ComponentNavBranch extends React.PureComponent<Props, State
     event.preventDefault();
     event.stopPropagation();
     event.currentTarget.blur();
-    this.setState({ open: true });
+    this.setState({ dropdownOpen: true });
   };
 
   closeDropdown = () => {
     if (this.mounted) {
-      this.setState({ open: false });
+      this.setState({ dropdownOpen: false });
     }
   };
 
-  render() {
+  togglePopup = (show?: boolean) => {
+    if (this.mounted) {
```

Comment:
Is it really useful to check to `mounted` here ? This method is not used asynchronously right ?